client/student/components/Home/index.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="columns is-centered">
+    <div class="column is-2">
+      <h1 class="subtitle has-text-centered">Select your Program:</h1>
+      <ul>
+        <li
+          v-for="it in programs"
+          :key="it.id"
+          @click="selectProgram(it.cohortId)">
```

Comment:
You can do it route based and selected program can be a getter (so you can omit this mutation). I know we initially talked about different approach but this might be nicer.